tests/queries/0_stateless/02404_memory_bound_merging.reference

```diff
@@ -113,12 +113,18 @@ ExpressionTransform
             (Expression)
             ExpressionTransform × 4
               (MergingAggregated)
-              Resize 1 → 4
-                SortingAggregatedTransform 4 → 1
-                  MergingAggregatedBucketTransform × 4
-                    Resize 1 → 4
-                      GroupingAggregatedTransform 3 → 1
-                        (ReadFromRemoteParallelReplicas)
+              MergingAggregatedBucketTransform × 4
```

Comment:
Check with @nickitat that the generated plan is correct

Reply:
I'm not sure if reading from the table with only a single stream is correct. but in general it looks right and it is different from the previous, because MBM wasn't supported, because we didn't have local plan
https://github.com/ClickHouse/ClickHouse/issues/63203